parse_logfwd_log.py

```
@@ -0,0 +1,15 @@
+import urllib.request
+import json
+import zipfile
+import io
+import os
+
+url = "https://api.github.com/repos/strawgate/memagent/actions/runs/24439395673/artifacts"
+req = urllib.request.Request(url)
+req.add_header("Authorization", f"Bearer {os.environ.get('GITHUB_TOKEN', '')}")
+with urllib.request.urlopen(req) as response:
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)